packages/design-system/src/components/DateField/SingleInputDateField.tsx

```diff
@@ -79,6 +79,11 @@ export interface SingleInputDateFieldProps extends FormFieldProps {
   toYear?: number;
 }
 
+/**
+ * For information about how and when to use this component, refer to the
+ * [Single Input Date Field](https://design.cms.gov/components/date-field/single-input-date-field/) and
+ * [Calendar Picker](https://design.cms.gov/components/date-field/date-picker/) documentation pages.
+ */
```

Comment:
In SB, there seems to be a props table nested inside a props table. `updatedValue` and `formattedValue` are nested inside `onChange`.

Reply:
Oh whoa! It's because we have the parameters of the functions listed in that prop's jsdoc-style comments. It looks a little weird, but it's actually helpful additional information in my opinion. I wish we had that all along